deltachat-ios/Controller/AudioRecorderController.swift

```diff
@@ -46,11 +46,14 @@ class AudioRecorderController: UIViewController, AVAudioRecorderDelegate {
         view.translatesAutoresizingMaskIntoConstraints = false
         view.alpha = 0.0
         view.contentMode = UIView.ContentMode.scaleAspectFit
+        view.isAccessibilityElement = true
```

Comment:
would be nice to get that piece  of code in though. I can also do a separate PR for it.